### ISSUE TITLE
Press `alt` to send keyboard focus to the menu bar 

### ISSUE THREAD

**Issue by Daniel-Chin** [godotengine/godot-proposals#4830]
### Describe the project you are working on

This is a general suggestion to the Godot editor. 

### Describe the problem or limitation you are having in your project

Currently, I cannot open "project settings" using *keyboard only*. 

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

In most Windows softwares, pressing `alt` makes the menu bar (i.e. "Scene" "Project" "Debug" "Editor" "Help" in Godot) acquire keyboard focus. Then, I can use arrow keys or letter keys to select menu items. The Godot editor should do the same. 

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

...

### If this enhancement will not be used often, can it be worked around with a few lines of script?

Maybe with `tool`? 

### Is there a reason why this should be core and not an add-on in the asset library?

- This is an accessibility feature and should be core. 
- It is lightweight. 
- Its existence doesn't hurt anything around.

**Comment by bruvzg** [comment]
> Does this kind of behavior exist on other platforms?

Not on macOS, it's <kbd>Command</kbd>+<kbd>Shift</kbd>+<kbd>/</kbd> instead (rebindable like most shortcuts).

**Comment by Daniel-Chin** [comment]
Prioritize #1701
